pkg/mounter/safe_mounter_windows.go

```diff
@@ -238,26 +237,31 @@ func (mounter *CSIProxyMounter) FindDiskByLun(lun string) (diskNum string, err e
 	// If match is found then return back the disk number.
 	for diskID, location := range findDiskByLunResponse.DiskLocations {
 		if strings.EqualFold(location.LUNID, lun) {
-			return diskID, nil
+			return strconv.Itoa(int(diskID)), nil
```

Comment:
nit: small utility func for the conversion.

Reply:
which part, the for loop?

Reply:
btw this FindDiskByLun function is copy/pasted to satisfy mounter interface , we don't use it anywhere.